wordproof-timestamp/includes/Page/SettingsPage.php

```diff
@@ -12,19 +12,12 @@ class SettingsPage {
 
     public function __construct() {
         add_action('admin_menu', array($this, 'addSettingsPage'));
-	    add_action('admin_init', array($this, 'registerSettings'));
-    }
-
-    public function registerSettings() {
-        register_setting( 'wordproof-options', 'wordproof_network' );
-        register_setting( 'wordproof-options', 'wordproof_store_content' );
-        register_setting( 'wordproof-options', 'wordproof_store_ram' );
-        register_setting( 'wordproof-options', 'wordproof_certificate_text' );
+	    add_action('admin_post_wordproof_form_action', array($this, 'saveSettings'));
     }
 
     public function addSettingsPage() {
         add_menu_page(
-            'WordProof',
+            __( 'WordProof', 'wordproof-timestamp' ),
```

Comment:
I see `'wordproof-timestamp'` being used quite often. Maybe make a constant out of this or wrap the function `__`?